deny.toml

```diff
@@ -1,14 +1,15 @@
+[graph]
 exclude=[
     "actix-http",
     "actix-http-tracing",
     "actix-udp",
     "actix-udp-example",
     "tracing-grpc",
-    "http"
+    "http",
```

Comment:
why http?

Reply:
I didn't add it, but it was added 3 years ago. I'll check tomorrow, which of them is still relevant 🧐

Reply:
oh okay. most can be cleaned up now!

Reply:
you were right, all of them didn't have to be excluded anymore 😅